tests/integration/providers/ldap/delta_outbound_sync_tests.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2018 Contributors to Hyperledger Sawtooth
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------------
+"""LDAP Delta Outbound Sync Tests"""
+# pylint: disable=redefined-outer-name
+# NOTE: disabling for pytest as per:
+# https://stackoverflow.com/questions/46089480/pytest-fixtures-redefining-name-from-outer-scope-pylint
+
+import pytest
+
+import rethinkdb as r
+
+from rbac.providers.common.db_queries import connect_to_db, peek_at_queue
+
+OUTBOUND_ENTRY_CASES = [
+    (
+        {
+            "data": {
+                "description": "The role that keeps on rolling",
+                "members": [],
+                "remote_id": "CN=Rolling_Role,OU=Roles,OU=Security,OU=Groups,DC=AD2012,DC=LAB",
+            },
+            "data_type": "group",
+            "timestamp": r.epoch_time(1376074395.012).to_iso8601(),
+            "provider_id": "test_provider",
+            "status": "UNCONFIRMED",
+            "action": "",
+        },
+        {
+            "data": {
+                "description": "The role that keeps on rolling",
+                "members": [],
+                "remote_id": "CN=Rolling_Role,OU=Roles,OU=Security,OU=Groups,DC=AD2012,DC=LAB",
+            },
+            "data_type": "group",
+            "timestamp": "2013-08-09T18:53:15.012+00:00",
```

Comment:
Is there any noticable difference between using a string literal like this and creating a new timestamp like `r.epoch_time(1376074395.012).to_iso8601()`?

Reply:
Accident, the field should be a datetime object.